tools/packaging/kata-deploy/local-build/Makefile

```diff
@@ -24,6 +24,8 @@ BASE_TARBALLS = serial-targets \
 	kernel-dragonball-experimental-tarball \
 	kernel-nvidia-gpu-tarball \
 	kernel-nvidia-gpu-confidential-tarball \
+	kernel-headers-nvidia-gpu-tarball \
+	kernel-headers-nvidia-gpu-confidential-tarball \
```

Comment:
Are there situations where you want only the headers, but not the kernel itself?

I'm asking here because, IMHO, we could *always* make sure that building the nvidia-gpu kernels we'd generate the deb and headers, and *always* add them to the ORAS container related to the kernel itself.  No need to add new extra targets.

WDYT?

Reply:
https://github.com/kata-containers/kata-containers/blob/e5e0983b56534f007cf06ab39f323961b17ff642/tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh#L185-L187

Here's what you can use, the extra_tarballs.